### PR TITLE
BuddyChat Individual Rooms

### PR DESCRIPTION
**Pull request by ccaldwell11**
Made updates to code to allow:

- Individual chat rooms between specific users
- Logic to identify sender/receiver for each user
      * Fixed my attempt to differentiate and visually distinguish between sent and received messages

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,19 +37,24 @@
...
     "@types/passport": "^1.0.16",
     "@types/passport-google-oauth20": "^2.0.14",
     "@types/react": "^18.2.74",
     "axios": "^1.6.8",
+    "cors": "^2.8.5",
     "dotenv": "^16.4.5",
     "express": "^4.19.2",
     "express-session": "^1.18.0",
     "lodash": "^4.17.21",
+    "nodemon": "^3.1.0",
```
Comment (peytono) [inline package.json]: Are you using nodemon anywhere?

File: src/context/UserContext.tsx [hunk 2]
```diff
@@ -6,6 +6,12 @@ export type AuthUser = {
   id: number,
   name: string,
   googleId: string,
+  buddies: Buddy[];
+};
+
+export type Buddy = {
+  id: number;
+  name: string;
```
Comment (peytono) [inline src/context/UserContext.tsx]: Does this have any weird effects on the user context? Is the current value of buddies in user context null?

File: package.json (hunk 1 above)
Comment (ccaldwell11) [reply package.json]: Good catch! I ended up not needing it.

File: src/context/UserContext.tsx (hunk 2 above)
Comment (ccaldwell11) [reply src/context/UserContext.tsx]: This doesn't have any weird effects but it is not being used right now. I commented it out right now until I get back around to using it or removing it if no longer needed.

Comment (peytono) [reply src/context/UserContext.tsx]: Sounds good!

**Review by peytono — APPROVED** [review]
Great work!

**Pull request merged**